### ISSUE TITLE
Evaluate the use of  MLA core repository for OAB (and if we already harvest it)

### ISSUE THREAD

**Issue by JosephMcArthur** [OAButton/discussion#203]

**Comment by JosephMcArthur** [comment]
They just got a big grant: https://news.commons.mla.org/2016/08/09/humanities-core-receives-neh-grant/

**Comment by JosephMcArthur** [comment]
This might now be called Humanities Commons, or, there is something called humanities commons we should look into.

**Comment by JosephMcArthur** [comment]
Okay, I played around with this a few nights ago. I couldn't get it to do a thing. Eventually, I got stuck in a redirect loop that was impossible to remove myself from. 

I saw no signs of a place to search or an API. This suggests to me that essentially there is nothing to see here for a while. Not going to close for now as one day there may be something. BUT, I'm going move this out the backlog for now.
